test/smart-wallet/onboardings-test.ts

```diff
@@ -492,7 +477,7 @@ describe("Wallet deployment cost estimation in various onbaording flows", functi
       buildContractCall(
         walletFactory,
         "deployCounterFactualWallet",
-        [owner, entryPoint.address, handler.address, 1],
+        [owner, 1],
```

Comment:
why 1 in this test?

Reply:
I would suggest storing index in a var/const with descriptive name

Reply:
sometime we need a wallet that's not already deployed as part of current test block execution.

Reply:
can do.

Reply:
done. const/var with comment beside